learn/building_programs.md

```diff
@@ -0,0 +1,780 @@
+---
+layout: page
+title: Building programs
+permalink: /learn/build_programs
+navbar: Learn
+---
+
+
+
+Draft tutorial: Building programs
+---------------------------------

```

Comment:
```suggestion
```
This won't be needed, the `title` field in the header will be rendered first as a H1 heading.